src/pages/HomePage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const HomePage: React.FC = () => (
+  <div className="section">
+    <div className="container">
```

Comment:
DRY. Now you use these 2 tags in every page component, move it to parent component